src/util/Options.js

```diff
@@ -69,6 +69,10 @@
  * [User Agent](https://discord.com/developers/docs/reference#user-agent) header
  * @property {PresenceData} [presence={}] Presence data to use upon login
  * @property {IntentsResolvable} intents Intents to enable for this connection
+ * @property {number} [fetchGuildTimeout=15000] Time in milliseconds that clients with the `GUILDS` intent should hang
+ * and wait for missing guilds to be received before starting the bot
+ * If not specified, the default is 15 seconds.
```

Comment:
Is this line really needed? The `[fetchGuildTimeout=15000]` in L72 will already show the default in the docs.